app/src/main/java/com/simple/checkstyledemo/MainActivity.java

```diff
@@ -2,6 +2,7 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.widget.Button;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.imports.UnusedImportsCheck
**Severity**: ERROR
**File**: app/src/main/java/com/simple/checkstyledemo/MainActivity.java L5

Unused import: android.widget.Button. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>*